controller_manager/src/ros2_control_node.cpp

```diff
@@ -33,36 +68,43 @@ int main(int argc, char ** argv)
 
   auto cm = std::make_shared<controller_manager::ControllerManager>(executor, manager_node_name);
 
-  // TODO(anyone): Due to issues with the MutliThreadedExecutor, this control loop does not rely on
```

Comment:
I think it would actually be wrong to use MutliThreadedExecutor even if it offered better performance in this case because it would use threads of the same priority to handle ROS callbacks as the thread used to service this loop which should be a higher priority.